clusterdock/cli.py

```diff
@@ -181,10 +185,12 @@ def main():
     if hasattr(args, 'topology'):
         # As a workaround for https://github.com/docker/for-mac/issues/2396, we write the /etc/localtime
         # into the clusterdock config directory, which can be mounted into containers.
-        if not os.path.exists(CLUSTERDOCK_CONFIG_DIRECTORY):
```

Comment:
may i suggest moving the config directory check/create logic outside of topology. so that way, if other actions which require config directory, it will be available.